### ISSUE TITLE
Exported Shapefile appears to be corrupted

### ISSUE THREAD

**Issue by lin-d-hop** [DigitalCommons/land-explorer-front-end#105]
**Describe the bug**

The Shapefile download is not creating a valid file download.

**To Reproduce**
1. Ensure map is saved.
2. Click Share -> Export -> Shapefile
3. An empty file downloaded successfully. Once populated the On subsequent attempts the file did not. Unsure if this is relevant.

**Expected behaviour**
Each time I change maps or edit the map a Shapefile should download with a legitimate shapefile zip.

**Screenshots/GIFs**
Errors:
![Screenshot from 2022-10-05 13-45-30](https://user-images.githubusercontent.com/8669848/194065399-dd2252c1-e802-4706-872d-8849f187dda9.png)
![Screenshot from 2022-10-05 13-55-00](https://user-images.githubusercontent.com/8669848/194065418-af901e46-0dab-49a4-98a5-35a03dd63631.png)

Example of corrupted file:[testshape-shapefile (2).zip](https://github.com/DigitalCommons/land-explorer-front-end/files/9715712/testshape-shapefile.2.zip)

**Context (please complete the following information):**
 - Device: [e.g. iPhone6] Laptop
 - OS: [e.g. iOS8.1] Ubuntu
 - Browser [e.g. stock browser, safari] CHrome
 - Version [e.g. 22] Production on Oct 5th

**Additional context**

**Comment by lin-d-hop** [comment]
Testing Notes:
1. I opened a saved map.
2. I downloaded the shapefile. 
3. Shapefile downloads perfectly and opens when tested [here](https://mapshaper.org/) :heavy_check_mark: 
Then
4. I created a new map
5. Turned on some layers
6. Saved the map
7. Tried to export the Shapefile
8. No file downloaded :x: 

I refreshed and tried again. Same problem.

Back to In Dev. Ping @rogup

**Comment by rogup** [comment]
I'm not able to download the shapefile at all in production, even on the first time.

It's working for me in dev, which has been confusing... but just had a look at the logs on production and can now see that pm2 is restarting the BE, since the shapefiles are temporarily stored in the project folder!

@King-Mob Is there a reason for the "--watch" flag in the 'npm run serve' command? I think it would be best to remove it and do a pm2 reload after pulling changes, but we could exclude watching the shapefiles directorry if there's a reason to keep it.

Also, do you have a link to the release guide, which outlines the steps for pulling changes and deploying them on the testing/production servers? I remember seeing it once but can't find it.

**Comment by rogup** [comment]
@King-Mob Bumping this in case it was missed

**Comment by King-Mob** [comment]
@rogup the reason for --watch being there was just to take out the need for restarting the app when building it. I think it's fine to just add that to the deploy instructions

**Comment by rogup** [comment]
Done, and the deployment instructions already have a pm2 restart step

**Comment by lin-d-hop** [comment]
Hi again @rogup 
Sorry to do this but it seems that the latest version of this has undone this PR.
Steps:
1) I turn on an RGS data layer, 
2) draw a poly, 
3) save the map, 
4) export the map 
5) See the RGS data is missing from the shapefile. The poly data is there though.

This seems like something unit tests would have caught ;)
Unfortunately this one is back to in dev.

**Comment by lin-d-hop** [comment]
[mapity-shapefile.zip](https://github.com/DigitalCommons/land-explorer-front-end/files/10508452/mapity-shapefile.zip)

**Comment by rogup** [comment]
@lin-d-hop I've tested this out on Mapshaper, and I'm seeing the same warning too, which is weird since I didn't see it last time.

However, the data is still visible for me and seems to contain all the data as expected, like names and descriptions. Are you sure that point objects are visible (they were off by default and I had to click the eye icon). And is the map zoomed out enough (clicking Basemap helps with this)?

![image](https://user-images.githubusercontent.com/48825436/214910754-f39e11d3-f101-4c7a-8c48-08a267f6f24f.png)
